firmware/common2015/utils/logger/logger.cpp

```diff
@@ -18,6 +18,24 @@ uint8_t rjLogLevel;
 Mutex log_mutex;
 
 /**
+ * [Collects the stream log message into a single string to print]
```

Comment:
I'd remove this comment since you have a copy of it in the header file already.  The two comments will inevitably get out of sync if anything changes.